apps/ledger-live-desktop/src/renderer/screens/platform/v2/Catalog/Card/hooks.ts

```diff
@@ -3,8 +3,10 @@ import { PropsRaw } from "./types";
 
 export function useCard({ manifest, onClick: onClickProp }: PropsRaw) {
   const hostname = useMemo(
-    () => new URL(manifest.url).hostname.replace("www.", ""),
-    [manifest.url],
+    // @ts-expect-error not all params type has dappUrl field
```

Comment:
is there a way to avoid the introduction of `ts-expect-error`?

Reply:
Yes, if all union types implements type guards for each pattern.

Reply:
Added a typeguard to prevent this issue

Reply:
Thanks @ComradeAERGO . CI looks fine for this PR. So, we can merge as soon as we get an approval.